modules/test-utils/src/main/java/org/jboss/wsf/test/JBossWSTest.java

```diff
@@ -304,90 +303,70 @@ public static void assertEquals(Element expElement, Element wasElement)
    public static void assertEquals(Object exp, Object was)
    {
       if (exp instanceof Object[] && was instanceof Object[])
-         assertEqualsArray((Object[])exp, (Object[])was);
+         assertArrayEquals((Object[])exp, (Object[])was);
       else if (exp instanceof byte[] && was instanceof byte[])
-         assertEqualsArray((byte[])exp, (byte[])was);
+         assertArrayEquals((byte[])exp, (byte[])was);
       else if (exp instanceof boolean[] && was instanceof boolean[])
-         assertEqualsArray((boolean[])exp, (boolean[])was);
+         assertArrayEquals((boolean[])exp, (boolean[])was);
       else if (exp instanceof short[] && was instanceof short[])
-         assertEqualsArray((short[])exp, (short[])was);
+         assertArrayEquals((short[])exp, (short[])was);
       else if (exp instanceof int[] && was instanceof int[])
-         assertEqualsArray((int[])exp, (int[])was);
+         assertArrayEquals((int[])exp, (int[])was);
       else if (exp instanceof long[] && was instanceof long[])
-         assertEqualsArray((long[])exp, (long[])was);
+         assertArrayEquals((long[])exp, (long[])was);
       else if (exp instanceof float[] && was instanceof float[])
-         assertEqualsArray((float[])exp, (float[])was);
+         assertArrayEquals((float[])exp, (float[])was);
       else if (exp instanceof double[] && was instanceof double[])
-         assertEqualsArray((double[])exp, (double[])was);
+         assertArrayEquals((double[])exp, (double[])was);
       else
-         Assert.assertEquals(exp, was);
+         Assertions.assertEquals(exp, was);
+   }
+   protected static void assertTrue(String reason, boolean condition) {
+      Assertions.assertTrue(condition, reason);
    }
 
-   private static void assertEqualsArray(Object[] exp, Object[] was)
-   {
-      if (exp == null && was == null)
-         return;
+   protected static void assertFalse(String reason, boolean condition) {
+      Assertions.assertFalse(condition, reason);
+   }
 
-      if (exp != null && was != null)
-      {
-         if (exp.length != was.length)
-         {
-            fail("Expected <" + exp.length + "> array items, but was <" + was.length + ">");
-         }
-         else
-         {
-            for (int i = 0; i < exp.length; i++)
-            {
+   protected static void assertEquals(String message, char expected, char actual) {
+      Assertions.assertEquals(expected, actual, message);
+   }
 
-               Object compExp = exp[i];
-               Object compWas = was[i];
-               assertEquals(compExp, compWas);
-            }
-         }
-      }
-      else if (exp == null)
-      {
-         fail("Expected a null array, but was: " + Arrays.asList(was));
-      }
-      else if (was == null)
-      {
-         fail("Expected " + Arrays.asList(exp) + ", but was: null");
-      }
+   protected static void assertEquals(String message, String expected, String actual) {
```

Comment:
thanks for the comment, I am good with removing these method wrappers. Yes, these were done for convenient to avoid rewriting assert methods in many places, but I get your point of misleading, will update it to use fully junit5 style only.